src/Catalog/Registration/RegistrationMakerCatalogItem.cs

```diff
@@ -155,6 +157,41 @@ private Uri GetPackageContentAddress()
             return _packageContentAddress;
         }
 
+        private string GetLicenseUrl()
```

Comment:
I didn't see UTs for this method

Reply:
The unit tests for this class are not very clear. I will discuss with Joel offline on Monday. Thanks!

Reply:
Add the unit tests now.